diff --git a/kernel/ptrace.c b/kernel/ptrace.c
index 72b0b3e..bfabd90 100644
--- a/kernel/ptrace.c
+++ b/kernel/ptrace.c
@@ -230,7 +230,7 @@ static int __ptrace_may_access(struct task_struct *task, unsigned int mode)
 	kgid_t caller_gid;
 
 	if (!(mode & PTRACE_MODE_FSCREDS) == !(mode & PTRACE_MODE_REALCREDS)) {
-		WARN(1, "denying ptrace access check without PTRACE_MODE_*CREDS\n");
+//		WARN(1, "denying ptrace access check without PTRACE_MODE_*CREDS\n");
 		return -EPERM;
 	}
 
